backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -107,12 +108,10 @@ export interface ExperimentCSVData {
   details: ConditionDecisionPointData[];
 }
 
-@EntityRepository()
-export class AnalyticsRepository {
-  constructor(private manager: EntityManager) {}
-
+@EntityRepository(AnalyticsRepository)
```

Comment:
This is inocrrect. Entity should be passes here and becuase this repository doesn't has any Entity nothing should be passed over here

Reply:
Okay, this change was made in older PR for release 5.2 typeorm changes. I will fix this. Thanks.